pkg/apis/enterprise/v1/common_types.go

```diff
@@ -146,6 +146,10 @@ type VolumeSpec struct {
 
 	// Remote Storage type.
 	Type string `json:"storageType"`
+
+	// App Package Remote Store provider.
+	// For e.g. AWS, Azure, Minio, etc.
```

Comment:
Should we list the currently supported types here as well?

Reply:
Yup sure, will do it.